### ISSUE TITLE
No exceptions are occurring, but the gestures defined in the config.gdef files are being triggered continuously -- even while the gesture shouldn't have been triggered. I suspect it has something to do with the EmulationController

### ISSUE THREAD

**Issue by jawaff** [WaffleTime/KinectController#18]

**Comment by jawaff** [comment]
The EmulationController seems to be working just fine. I changed the gesture definition I was debugging with. Apparently the problem was that I was using multiple Angle rules for the gesture. A single Angle rule for the gesture seems to work just as intended. I'll have to look into this a bit more.

**Comment by jawaff** [comment]
Figured it out! The Esper pattern I was using was pretty flawed. I ended up creating a separate repository on github for testing out the behavior of different patterns in a controlled environment (https://github.com/WaffleTime/EsperTest.)

My concluding Esper pattern is:
select "<InsertId>" as gestureId from pattern[every((~A or ~B) -> (A and B)) where timer:within(1 sec)]

This pattern is doing exactly what I was expecting the first one to do!
